pom.xml

```diff
@@ -9,8 +9,7 @@
     <version>2.2.0</version>
   </parent>
   <artifactId>core-utils-lib</artifactId>
-
-  <version>1.3.0-SNAPSHOT</version>
+  <version>1.5.1-SNAPSHOT</version>
```

Comment:
Is it possible to use `1.5.0` instead? Snapshot versions in the pom are dangerous.

Reply:
The current versioning on the development branch is 1.5.0-SNAPSHOT to which i wanted to merge the qc fix.  
As far as i understood it, we only remove SNAPSHOT when we push it to master branch.

Reply:
I just realized that this is not a dependency. It should still be `1.5.0-SNAPSHOT` since not `1.5.0` release has been made and your fix would be included in the `1.5.0` release. Correct?

Reply:
Ah good point! 
I forgot that we haven't released 1.5.0 yet. Will fix it! Thank you 👍